Tasks/JenkinsDownloadArtifacts/task.json

```diff
@@ -122,6 +163,28 @@
         "ServerCallFailed": "HTTP call to Jenkins server failed.",
         "ServerCallErrorCode": "HTTP response code: %d",
         "ArtifactSuccessfullyDownloaded": "Successfully downloaded artifact to %s",
-        "ArtifactDownloadUrl": "Downloading artifacts from: %s"
+        "ArtifactDownloadUrl": "Downloading artifacts from: %s",
+        "DownloadingCommitsAndWorkItems": "Downloading commits and work items",
+        "JenkinsDownloadingChangeFromCurrentBuild": "Cannot find start build, will be fetching commits and workitems from build used to download artifacts only: %s",
```

Comment:
Suggest rewording to "Cannot find start build, only commits and workitems from build %s will be fetched" @DavidStaheli can provide better wording.

Reply:
Changed to "Cannot find start build, commits and workitems will be downloaded from build: %s"